Deployment/check_service_status.ps1

```diff
@@ -0,0 +1,55 @@
+Param(
+	[Parameter(mandatory=$true)][string]$healthEndPointUrl,
+    [Parameter(mandatory=$true)][string]$waitTimeInMinute
+)
+
+$sleepTimeInSecond = 15
+$isServiceActive = 'false'
+
+$stopWatch = New-Object -TypeName System.Diagnostics.Stopwatch
+$timeSpan = New-TimeSpan -Minutes $waitTimeInMinute
+$stopWatch.Start()
+
+do
+{
+    Write-Host "Polling url: $healthEndPointUrl ..."
+    try{
+        Start-Sleep -Seconds $sleepTimeInSecond
+        $HttpRequest  = [System.Net.WebRequest]::Create("$healthEndPointUrl")
+        $HttpResponse = $HttpRequest.GetResponse() 
+        $HttpStatus   = $HttpResponse.StatusCode
+        Write-Host "Status code of web is $HttpStatus ..."
+    
+        If ($HttpStatus -eq 200 ) {
+            Write-Host "Service is up. Stopping Polling ..."
+            $isServiceActive = 'true'
+            break
+        }
+        Else {
+            Write-Host "Service not yet Up. Status code: $HttpStatus ..."
+            Start-Sleep -Seconds $sleepTimeInSecond
+        }
```

Comment:
It looks sleep block is repeated and code will wait for twice of `sleepTimeInSecond`. Please can you check.

```suggestion
        $HttpRequest  = [System.Net.WebRequest]::Create("$healthEndPointUrl")
        $HttpResponse = $HttpRequest.GetResponse() 
        $HttpStatus   = $HttpResponse.StatusCode
        Write-Host "Status code of web is $HttpStatus ..."
    
        If ($HttpStatus -eq 200 ) {
            Write-Host "Service is up. Stopping Polling ..."
            $isServiceActive = 'true'
            break
        }
        Else 
        {
            Write-Host "Service not yet Up. Status code: $HttpStatus ..."
         }
        Start-Sleep -Seconds $sleepTimeInSecond
```

Reply:
Removed sleep from all places and added at the end for loop.